frontend/modules/admin_portal/students_view/AdminStudents.js

```diff
@@ -281,6 +297,10 @@ export default React.createClass({
           alert("Error: [" +assnId+ "] grades must be integers between 0-100.");
```

Comment:
@rtholmes 
One edge case here is when no final grade has been previously assigned, you must enter it.
Once a final grade has been previously assigned, you are free to only update any other fields.
Let me know if this needs to be changed as well so that a final grade is not needed when you initially assign grades to a deliverable (compared to updating grades).